crates/quickjs-sys/Cargo.toml

```diff
@@ -3,6 +3,11 @@ name = "quickjs-sys"
 version = "0.1.0"
 authors = ["Saúl Cabrera <saulecabrera@gmail.com>"]
 edition = "2018"
+license = "Apache-2.0"
```

Comment:
> Should we switch this to be MIT?

I can.

> I'm not sure if the build dependencies count towards the final licensing of the crate

They can. Consuming a header file as part of a build process makes the resulting artifact a derived work. That said, we'd still be safe to license this crate as MIT since we're no longer vendoring the wasi-sdk code. The Apache license only applies to source code that was licensed with an Apache license, not to derived works.